instrumentation/spring/spring-security-config-6.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.springframework.security")
+    module.set("spring-security-config")
+    versions.set("[6.0.0,]")
+
+    extraDependency("jakarta.servlet:jakarta.servlet-api:6.0.0")
+    extraDependency("org.springframework.security:spring-security-web:6.0.0")
+    extraDependency("io.projectreactor:reactor-core:3.5.0")
+  }
+}
+
+dependencies {
+  implementation(project(":instrumentation:spring:spring-security-config-6.0:library"))
+  implementation("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
```

Comment:
usually we'd use `project(":javaagent-extension-api")` I think this dependency is added automatically, you shouldn't need to add it manually.

Reply:
removed